default/scripting/species/common/telepathic.macros

```diff
@@ -4,9 +4,11 @@ TELEPATHIC_DETECTION
             scope = And [
                 PopulationCenter
                 Not Population high = 0
-                WithinStarlaneJumps jumps = @1@ condition = Source
                 Not VisibleToEmpire empire = Source.Owner
+                Not OwnedBy empire = Source.Owner
+                WithinStarlaneJumps jumps = @1@ condition = Source
             ]
-            activation = OwnedBy affiliation = AnyEmpire
+            // Activate the turn after source creation to allow regular vision to prevail
+            activation = Turn low = Source.CreationTurn + 1
```

Comment:
What happens if a planet has a telepathic species added to it by an effect and has its ownership set on the same turn so that an empire should get vision of a planet for the first time on the same turn it has that species, but it's not the planet's creation turn?